packages/core/test/workflow/buffer-reducing.test.js

```diff
@@ -123,8 +123,8 @@ test('can reduce received buffers by creating an aggregate and remaining buffer'
 
   const bucketName = await createBucket(s3)
   const { buffers, bufferRecords } = await getBuffers(2, {
-    length: 100,
-    size: 128
+    length: 10,
```

Comment:
just changed to use same values as integration tests and be able to catch potential issues faster there